pkg/ebpf/uprobes/inspector.go

```diff
@@ -129,3 +127,21 @@ func (*NativeBinaryInspector) symbolToFuncMetadata(elfFile *elf.File, sym elf.Sy
 func (*NativeBinaryInspector) Cleanup(_ utils.FilePath) {
 	// Nothing to do here for the native inspector
 }
+
+// SanitizeAddresses - sanitizes the addresses of the provided symbols
+func SanitizeAddresses(f *safeelf.File, syms []safeelf.Symbol) {
```

Comment:
instead of re-implementing the function, you should be declaring `safeelf.Symbol` as an alias to `elf.Symbol`
it will allows us to use the original implementation of `manager.SanitizeUprobeAddresses`